AGENTS.md

```
@@ -0,0 +1 @@
+@.github/copilot-instructions.md
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)